test/e2e/servers/graphql-main-server/no-auth-requests.spec.ts

```diff
@@ -38,8 +39,10 @@ const { phone, code } = getPhoneAndCodeFromRef("G")
 
 beforeAll(async () => {
   await initializeTestingState(defaultStateConfig())
-  correctCode = `${code}` as PhoneCode
   serverPid = await startServer("start-main-ci")
+  await loginFromPhoneAndCode({ phone, code })
```

Comment:
you are not using client, what is the purpose of this?

Reply:
to create the user/account in the db